components/alert/alert.js

```diff
@@ -127,12 +152,16 @@ class Alert extends LocalizeStaticMixin(RtlMixin(LitElement)) {
 	render() {
 		return html`
 			<div class="d2l-alert-highlight"></div>
-			<div class="d2l-alert-text">
+			<div class="d2l-alert-text d2l-body-standard">
 				<slot></slot>
 				${this.subtext ? html`<p class="d2l-body-compact d2l-alert-subtext">${this.subtext}</p>` : null}
 			</div>
-			${this.buttonText && this.buttonText.length > 0 ? html`<d2l-button-subtle class="d2l-alert-action" text=${this.buttonText} @click=${this._onButtonClick}></d2l-button-subtle>` : null}
-			${this.hasCloseButton ? html`<d2l-button-icon class="d2l-alert-action" icon="d2l-tier1:close-default" text="${this.localize('close')}" @click=${this.close}></d2l-button-icon>` : null}
+			<div class="d2l-alert-action">
```

Comment:
So by always wrapping these with `d2l-alert-action`, I noticed that when an alert doesn't have a close button or buttonText, there's an extra `40px` of space taken up by this container.

Also, the spec says that the text container should have 30px (1.5rem) of padding around it, but on the right it only has 24px currently.

Since the padding of the `d2l-alert-text` container is conditional on there being actions or not, I think I'd use conditional classes. So like `d2l-alert-text` gives it 1.5rem of padding but then it would add `d2l-alert-text-with-actions` if there are actions that would reduce the right padding.

To simplify things, I might also suggest positioning the coloured band absolutely -- that way it isn't taking up any actual space, it would just float on top.